spec.md

```diff
@@ -560,13 +549,294 @@ schema being used.
         "bullets": [
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$ref": "osb.v2:///fakeservice/v1#BillingAccountParameters"
+            }
+          },
+          "update": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "propeties": {
+                "some-other-account": {
+                  "description": "Other fake account needed for update.",
+                  "type": "string"
+                },
+                "billing": {
+                  "$ref": "osb.v2:///fakeservice/v1#BillingAccountParameters"
+                }
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "create": {
+            "parameters": {
+              "$ref": "osb.v2:///fakeservice/v1/bindings#AccountParameters"
+            }
+          }
+        }
       }
     }]
   }]
 }
 ```
 
 
+## JSON Schemas
+
+The JSON Schemas endpoint is used to define reusable definitions of JSON Schema
+Objects referenced throughout the [Catalog](#catalog-management). If a Platform
+intends to support JSON Schemas, it MUST examine the response from the catalog
```

Comment:
does it have a choice in the matter?

Reply:
By that I mean, the Platform could choose to support schema but it can't choose to not support the schema endpoint if it does support schema - correct?

Reply:
That is correct. If the platform supports JSON Schema with `$ref`, then it must support the schemas endpoint.

Reply:
This requirement may affect adoption of schemas in general. because schemas are already in spec, this addition will result in making platforms that are compliant (and support schemas as of v2.12) non-compliant.

Could versioning, or some other signal, be used for platform to communicate to the broker: "I like schemas, but can't yet deal with references and schema endpoint", and broker can either inline schemas, or use $refs to its schema endpoint?

Reply:
The api version is already in every request. The broker could implement this if they choose to, but it is up to that broker.